<?xml version="1.0" encoding="UTF-8" ?>
<runHistory script="connect">
 <run>
  <RunDate>2023-04-18 13:32:32.373 +0900</RunDate>
  <Status>Failure</Status>
  <Duration>0.699000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20230418_133232.373/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STMaxCallDepthExceeded -- Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself.</ErrorMessage>
 </run>
 <run>
  <RunDate>2023-04-18 13:32:54.920 +0900</RunDate>
  <Status>Failure</Status>
  <Duration>0.530000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20230418_133254.920/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STMaxCallDepthExceeded -- Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself.</ErrorMessage>
 </run>
 <run>
  <RunDate>2023-04-18 13:35:17.352 +0900</RunDate>
  <Status>Failure</Status>
  <Duration>0.522000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20230418_133517.352/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STMaxCallDepthExceeded -- Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself.</ErrorMessage>
 </run>
 <run>
  <RunDate>2023-04-18 13:38:49.650 +0900</RunDate>
  <Status>Failure</Status>
  <Duration>0.448000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20230418_133849.650/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STMaxCallDepthExceeded -- Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself.</ErrorMessage>
 </run>
</runHistory>
